js/src/widgets/hud.js

```diff
@@ -293,6 +334,11 @@
                                   '<i class="material-icons">crop</i>',
                                   '</a>',
                                   '</div>',
+                                  '<div class="mirador-download-control">',
+                                  '<a class="mirador-download-button hud-control" role="button" title="{{t "downloadButton"}}" aria-label="{{t "downloadButton"}}" download="image">',
```

Comment:
Since Firefox will only honor the `download` attribute when the image and the script have the same origin, add a `target="_blank"` attribute to the link so that Firefox will at least open a new tab instead of just following the link and losing the user's place in the Mirador viewer.